### ISSUE TITLE
Content length counting

### ISSUE THREAD

**Issue by k18923** [kevinfaguiar/vue-twemoji-picker#56]
Hi, 
I want to count content length exactly with the [Twitter rule](https://developer.twitter.com/en/docs/basics/counting-characters).
Twitter accepts 280 characters in a tweet, but 140 characters with Emojis(and [Japanese](https://developer.twitter.com/ja/docs/basics/counting-characters), Chinese, Korean characters). 

Emojis are counted 1 character on Twemoji-textarea with [maxlength](https://kevinfaguiar.github.io/vue-twemoji-picker/docs/twemoji-textarea-api/#maxlength
) prop, but are counted 2 characters on Twitter.

### example
140 Emojis

* Twemoji-textarea
![image](https://user-images.githubusercontent.com/15007826/79035813-d690a480-7bfc-11ea-8e29-218caccea18c.png)

* Twitter
![image](https://user-images.githubusercontent.com/15007826/79035971-4d7a6d00-7bfe-11ea-9827-719d256c7275.png)

### suggestion
I suggest that to use [twitter-text](https://github.com/twitter/twitter-text) instead of `Array.length` below,
https://github.com/kevinfaguiar/vue-twemoji-picker/blob/7094fb244e94a844b9b24fb1b42b1b6fb173e71d/src/components/TwemojiTextarea.vue#L222

``` javascript
this.actualContentLength = twitter.parseTweet(content).weightedLength;
```

Or create a new maxlength prop for twitter rule.

**Comment by kevinfaguiar** [comment]
Hi there @k18923! I will study your suggestion but in advance I would like to thank you for your help, seems like a really good library for parsing. Will study it a lil more however haha

**Comment by kevinfaguiar** [comment]
I really liked the 'twitter-text' library and this post will serve as an explanation of "why" of the change I've made in version 5.6.3. From this version into the future the length of each emoji will be weighted as 2. I think Twitter as a good example of heavy usage of emoji features in software development and would be not bad to use their lib 'twitter-text'.
 
That said, it is now available from version 5.6.3 onwards and on the docs: https://kevinfaguiar.github.io/vue-twemoji-picker/docs/twemoji-textarea-api/#actualcontentlengthchanged
